frontend/www/js/omegaup/arena/contest_practice.ts

```diff
@@ -26,7 +28,7 @@ OmegaUp.on('ready', () => {
           problems: this.problems,
           problemInfo: this.problemInfo,
           problem: this.problem,
-          activeTab: 'problems',
+          activeTab: this.activeTab,
```

Comment:
```suggestion
          activeTab,
```

cambiando L14 por
```typescript
// ... el mismo comentario del otro typescript de esta semana o la pasada.
const activeTab = window.location.hash ? window.location.hash.substr(1).split('/')[0] : 'problems';
```

este prop no se está cambiando después de cargar la página, entonces no hay necesidad de ponerlo en `data`.

Reply:
Cierto, ya lo arreglé